### PR TITLE
Adds Danger

### LINKED ISSUE #6: Add Danger/SwiftLint
As we have [CI](https://github.com/Moya/Harvey/pull/1) now, it would be awesome to have these checks. We can copy current ones from Moya.

### PR DESCRIPTION
**Pull request by ashfurrow**
Fixes #6.

### CODE REVIEW

File: Dangerfile [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+# Changelog entries are required for changes to library files.
+no_changelog_entry = !git.modified_files.include?("Changelog.md")
+if has_app_changes && no_changelog_entry && not_declared_trivial
+  warn("Any changes to library code should be reflected in the Changelog. Please consider adding a note there and adhere to the [Changelog Guidelines](https://github.com/Moya/contributors/blob/master/Changelog%20Guidelines.md).")
+end
+
+podspec_updated = !git.modified_files.grep(/Harvey.podspec/).empty?
+package_updated = !git.modified_files.grep(/Package.swift/).empty?
+
+if !(a^b) # an inverted xor, I had to look it up
```
Comment (SD10) [inline Dangerfile]: are `a` and `b` undefined here? Meant to reference other variables possibly?

Comment (ashfurrow) [reply Dangerfile]: haha, oops. Forgot to change names when I copied from irb 😅

**Comment by ashfurrow** [comment]
This is good to go 👍

**Comment by orta** [comment]
If you can live without Swift Lint for now, you could use Danger Swift instead?

**Comment by sunshinejr** [comment]
Oh, that would be cool! Seems like you've already have the work almost finished, I'm up for a swifty version, swiftlint will come probably really fast 😉

**Pull request merged**